qcfractal/interface/client.py

```diff
@@ -241,15 +245,37 @@ def add_options(self, opt_list, full_return=False):
 
     ### Collections section
 
-    def get_collections(self, collection_list, full_return=False):
+    def list_collections(self, collection_type=None):
```

Comment:
Thinking about de-duplication, can you have this function just call `get_collection` or have both of them call a hidden function?

Reply:
Duplicated 2-3 lines isn't worth writing an extra function IMO. Most of the deduplication is handled in the call function.